labs/b01/b01.md

```diff
@@ -0,0 +1,27 @@
+# Lab 01
+Welcome to the first lab! Here are some commands you might find helpful:
+`cat, cd, emacs, file, grep, less, ls, man, nano, tar, wget, vim`
+
+Remember: Google and `man` are your friends!
+
+## Shell spelunking
+**Everything should be done via the shell!**
+The purpose of this lab is to get you comfortable with using the shell for things you might typically use a GUI for. While these tasks may seem simplistic or limited, you'll quickly find that the commands have many different options (flags) to perform tasks that are either impossible or incredibly tedious / difficult to complete using traditional methods.
+
+0. Download `b01` from `https://decal.ocf.berkeley.edu/labs/b01/b01.tgz` and decompress it.
```

Comment:
They might need clues on how to do this? Some students may not know what compression/decompression even is.

Reply:
Also, I'm not so sure we should upload large binaries to git, it makes cloning the repo take way longer and pollutes the history. I'd put this in the decal user folder on NFS somewhere so it gets backed up but not necessarily in Git. Perhaps others will have more thoughts on the matter though.

Reply:
large binaries shouldn't be in git, but smaller files and supporting directory structures can be. they should be in the 0xcf/decal-labs repo though.

in this case, maybe just have b01.tgz in public_html somewhere so students can download it from there.

Reply:
one concern would be that the link could likely break after this semester though, so not sure what a great way to fix that would be unless we just change the filename to be b01-spring2018.tgz or something and leave it in some "static" directory forever

Reply:
we can always put it into /static/labs/sp18/ and use absolute links always, so it would be unlikely to break for the forseeable future.